src/platform/plugins/shared/data_views/server/rest_api_routes/internal/fields.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import { createHandler } from './fields';
+import { IndexPatternsFetcher } from '../../fetcher';
+import { coreMock, httpServerMock } from '@kbn/core/server/mocks';
+import { RequestHandlerContext } from '@kbn/core/server';
+
+jest.mock('../../fetcher');
+
+describe('handler', () => {
```

Comment:
there could be much more tests here, but just started with a test to catch the issue